### PR TITLE
Move basic dependencies install to requirements-ci

### PR DESCRIPTION
**Pull request by malfet**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #137995
* __->__ #138024
* #138023

### CODE REVIEW

File: .ci/docker/common/install_conda.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+!/bin/bash
```
Comment (huydhn) [inline .ci/docker/common/install_conda.sh:1]: Removing '#' is definitely a typo

**Review by huydhn — APPROVED** [review]
There is a typo in the PR that needs to be fixed, but the rest LGTM!

File: .ci/docker/common/install_conda.sh (hunk 1 above)
Comment (huydhn) [reply .ci/docker/common/install_conda.sh:1]: I'm temp to add this back, but then I realize that you're using ghstack :)

Comment (malfet) [reply .ci/docker/common/install_conda.sh:1]: Surprisingly it still works...

**Pull request closed without merging**